baselines/fedper/fedper/run_figures.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# CIFAR10 Mobile and Resnet (non-iid n classes (FIGURE 2a&b))
+for model in mobile resnet
+do 
+    for num_classes in 4 8 10
+    do
+        for algorithm in fedper fedavg
+        do
+           python -m fedper.main --config-path conf --config-name cifar10 dataset.num_classes=${num_classes} model_name=${model} algorithm=${algorithm}
+        done
+    done
+done
+
+
+# CIFAR10 Mobile (n head layers (FIGURE 4a))
+for num_head_layers in 2 3 4
```

Comment:
```suggestion
for num_head_layers in 1 2 3
```

Reply:
Figure 4a in the paper show 1,2,3?

Reply:
Hi Javier, 

the figure in the paper show 1,2,3 blocks + classifier. I count the classifier as a head layer, thus I put 2,3, and 4.